ddtrace/contrib/pymongo/client.py

```diff
@@ -285,6 +288,14 @@ def __trace(self, cmd):
             set_address_tags(s, self.address)
         return s
 
+    def validate_session(self, client, session):
```

Comment:
I need to work out how to get his override into `https://github.com/mongodb/mongo-python-driver/blob/master/pymongo/pool.py#L861`